packages/dialtone-vue2/components/emoji_picker/emoji_picker.vue

```diff
@@ -80,43 +81,121 @@ export default {
   },
 
   props: {
+    /**
+     * The array with recently used  emoji object
+     * This list is necessary to fill the recently used tab
+     * @type {Array}
+     * @default []
+     * @example
+     * <dt-emoji-picker :recentlyUsedEmojis="[emojiObject, emojiObject]" />
+     */
+    // TODO try to simplify this to achieve an array of unicode characters and not an entire emoji data object
     recentlyUsedEmojis: {
       type: Array,
     },
 
+    /**
+     * The placeholder text for the search input
+     * @type {String}
+     * @required
+     * @example
+     * <dt-emoji-picker :searchPlaceholderLabel="'Search...'" />
+     */
     searchPlaceholderLabel: {
       type: String,
       required: true,
     },
 
+    /**
+     * The label for the search results tab
+     * @type {String}
+     * @required
+     * @example
+     * <dt-emoji-picker :searchResultsLabel="'Search results'" />
+     */
     searchResultsLabel: {
       type: String,
       required: true,
     },
 
+    /**
+     * The label for the search no results
+     * @type {String}
+     * @required
+     * @example
+     * <dt-emoji-picker :searchNoResultsLabel="'No results'" />
+     */
     searchNoResultsLabel: {
       type: String,
       required: true,
     },
 
+    /**
+     * The list of tabsets to show, it is necessary to be updated with the correct language
+     * It must respect the provided order.
+     * @type {Array}
+     * @required
+     * @example
+     * <dt-emoji-picker
+     *  :tabSetLabels="['Most recently used', 'Smileys and people', 'Nature',
+     *    'Food', 'Activity', 'Travel', 'Objects', 'Symbols', 'Flags']" />
+     */
     tabSetLabels: {
       type: Array,
       required: true,
     },
 
+    /**
+     * The skin tone to show the emojis
+     * This prop gives the possibility to use the skin tone selected by the user previously
+     * @type {String}
+     * @default 'Default'
+     * @values 'Default', 'Light', 'MediumLight', 'Medium', 'MediumDark', 'Dark'
+     * @example
+     * <dt-emoji-picker :skinTone="'Default'" />
+     */
     skinTone: {
       type: String,
     },
 
+    /**
+     * Tooltip shown when skin selector button is hovered.
+     * @type {String}
+     * @required
+     * @example
+     * <dt-emoji-picker :skin-selector-button-tooltip-label="'Change default skin tone'" />
+     */
     skinSelectorButtonTooltipLabel: {
       type: String,
       required: true,
     },
+
+    /**
+     * Sets the search query that filters emojis.
+     * @type {String}
+     * @example
+     * <dt-emoji-picker search-query="smile" />
+     */
+    searchQuery: {
+      type: String,
+      default: '',
+    },
+
+    /**
+     * Shows the search input
+     * @type {String}
+     * @example
+     * <dt-emoji-picker :show-search="false" />
+     */
+    showSearch: {
+      type: Boolean,
+      default: true,
+    },
```

Comment:
Forgot to add this props to `emoji_picker.stories.js` storybook is not correctly setting the types to String and Boolean.

<img width="1432" alt="Screenshot 2024-05-31 at 2 06 50 p m" src="https://github.com/dialpad/dialtone/assets/87546543/b27c4f5f-db36-42e4-9420-3be25f873978">

Reply:
no idea why it's not inferring this automatically actually.. but I made the change.